### ISSUE TITLE
Files open in notification window

### ISSUE THREAD

**Issue by loqusion** [nvim-neo-tree/neo-tree.nvim#575]
When I try to open a file with `<CR>`, if there's a notification window visible then the file will open there. This happens when I use [noice.nvim](https://github.com/folke/noice.nvim) and it echoes regular output with `vim.notify`, then I quickly open Neotree and open a different file.

<img width="794" alt="スクリーンショット 2022-10-19 154428" src="https://user-images.githubusercontent.com/38332081/196800363-91a41f38-ef06-4e9e-965e-44cd284bc596.png">

**Comment by pysan3** [comment]
What is your nvim version and are you on the latest neo-tree?

I got this problem long ago but I think it got magically fixed somewhere around nvim v0.7.0 ~ v0.8.0.

**Comment by loqusion** [comment]
```
NVIM v0.8.0
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by brew@HMBRW-A-001-M1-004.local
```

I'm using the `v2.x` branch of neo-tree.

On a whim, I disabled [`stabilize.nvim`](https://github.com/luukvbaal/stabilize.nvim) and that fixed the issue. So I guess I should create an issue there, or possibly even [neovim/neovim](https://github.com/neovim/neovim) since neovim/neovim#19243 includes its functionality.

I'll go ahead and close this issue.
